app/modules/parsing/graph_construction/parsing_controller.py

```
@@ -237,6 +238,51 @@ def _on_email_done(t: asyncio.Task) -> None:
+                                            None,
+                                            lambda: _rm.prepare_for_parsing(
+                                                _rn, _ref, auth_token=_at, is_commit=_ic, user_id=_uid
+                                            ),
+                                        )
+                                        logger.info(
+                                            "Background worktree ensured for READY project %s (%s@%s)",
+                                            project_id,
+                                            _rn,
+                                            _ref,
+                                        )
+                                    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)